mkdocs/docs/admin-guide.md

```diff
@@ -269,17 +271,17 @@ result: 0 Success
 ```
   
 ## How to find Herd-MDL logs in CloudWatch
-Note: Logs are created in cloudwatch only when the log content is not empty, therefore, some of the log streams mentioned below may not be found if the logs are empty.
+Note: Logs are created in cloudwatch only when the log content is NOT empty, therefore; some of the log streams mentioned below may not be found if there is no content in the logs.
```

Comment:
therefore,

Reply:
right, I should've actually put the semicolon before 'therefore'